### ISSUE TITLE
hyper-parameter setting problem

### ISSUE THREAD

**Issue by cfzd** [Wizaron/instance-segmentation-pytorch#9]
The parameters DELTA_VAR and DELTA_DIST in discriminative loss function are set to 0.5 and 1.5 respectively for both CVPPP and cityscapes datasets. 

Is there any principle for these parameters or these two parameters are just empirical values ?
